### PR TITLE
Update readme listing more repos using the issue labeler

### PR DESCRIPTION
**Pull request by maryamariyan**

### CODE REVIEW

File: src/Microsoft.DotNet.GitHub.IssueLabeler/Controllers/PredictionEngineController.cs [hunk 1]
```diff
@@ -14,19 +14,19 @@
 
 namespace Microsoft.DotNet.GitHub.IssueLabeler
 {
-    [Route("api/WebhookIssue")]
-    public class WebhookIssueController : Controller
+    [Route("api/PredictionEngine")]
```
Comment (Eilon) [inline src/Microsoft.DotNet.GitHub.IssueLabeler/Controllers/PredictionEngineController.cs:17]: This is a breaking change that will break Hubbup. Maybe we can just _add_ the new route and have both enabled until I can migrate Hubbup to use the new URL?

Comment (maryamariyan) [reply src/Microsoft.DotNet.GitHub.IssueLabeler/Controllers/PredictionEngineController.cs]: Created https://github.com/dotnet/issue-labeler/pull/37

**Review by Eilon — APPROVED** [review]

File: Documentation/README.md [hunk 2]
```diff
@@ -34,36 +34,20 @@ As seen in [commit](https://github.com/dotnet/issue-labeler/commit/77e4dbc45184f
...
 Since dotnet/runtime has a big set of area owners and contributors, we decided to use an automatic assignemnt for issues and PRs. In order to achieve automatic label assignments, the IssueLabeler app, listens to all issue and PR creations via a webhook setting and finds top three predictions and only when the top prediction score has above 40% confidence, then this labeler app is allowed to automatically add that area label name to the newly created issue or PR. For dotnet/aspnetcore however, this webhook is not active and instead, the aspnetcore repository uses the hubbup web app to allow for manual area label assignment. Rather than doing automatic assignments, the hubbup app provides a nice UI for the prediction results it receives from [Microsoft.DotNet.GitHub.IssueLabeler](https://github.com/dotnet/issue-labeler/tree/master/src/Microsoft.DotNet.GitHub.IssueLabeler).
```
Comment (dakersnar) [inline Documentation/README.md:42]: Nit: assignemnt -> assignment

**Review by dakersnar — COMMENTED** [review]
Looks good! I'll look at this again when I start working with the project, and I'll let you know if I run into anything else that the documentation should reflect.

**Pull request merged**